crates/gpui/src/geometry.rs

```diff
@@ -2761,6 +2805,54 @@ impl Half for GlobalPixels {
     }
 }
 
+/// Provides a trait for types that can invert their values.
```

Comment:
Technically speaking, there is no such thing as "invert" for a type.
What you mean actually here is "invert" for a particular *operation* on a type, such as addition or multiplication. And in these implementations you assume a specific operation.
I think it's ambiguous, and the naming needs to be changed. For example, can we just use https://doc.rust-lang.org/stable/std/ops/trait.Neg.html instead?

Reply:
Good point there, should be negate.